modules/api/src/main/java/org/eclipse/xpanse/api/config/AuditLogWriter.java

```diff
@@ -109,6 +110,7 @@ private Csp getCspFromMethodInfo(AuditApiRequest auditApiRequest, Object[] args)
                 return csp;
             }
         } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
@baixinsui better we always stick to our logger so that the log is written with all MDC context necessary.